templates/corporate/legacy_server_login.html

```diff
@@ -0,0 +1,45 @@
+{% extends "zerver/portico.html" %}
+{% set entrypoint = "upgrade" %}
+
+{% set PAGE_TITLE = "Zulip server billing management" %}
+
+{% block portico_content %}
+
+<div class="register-account flex full-page server-login-page">
+    <div class="center-block new-style">
+        <div class="pitch">
+            <h1>Zulip server billing management</h1>
+        </div>
+        <div class="white-box">
+            {% if error_message %}
+            <div id="server-login-error" class="alert alert-danger">{{ error_message }}</div>
+            {% endif %}
+            <div id="server-login-input-section">
+                <form id="server-login-form" method="post" action="/serverlogin/">
+                    {{ csrf_input }}
+                    <div class="input-box server-login-form-field">
+                        <label for="username" class="inline-block label-title">
+                            server_org_id
+                            <a href="https://example.com" target="_blank">
+                                <i class="fa fa-question-circle-o" aria-hidden="true"></i>
+                            </a>
```

Comment:
This little question mark should open a page with a doc about where to grab this secret instead of example.com of course. Unless we already have something that can be plugged here?

Reply:
Let's hardcode https://zulip.readthedocs.io/en/stable/production/mobile-push-notifications.html for now and we can adjust that later in QA when we have a better anchor to link to.